subprojects/packagefiles/quazip/quazip/meson.build

```diff
@@ -28,15 +29,14 @@ quazip_src = [
   'zip.c'
 ]
 
-quazip_pp = qt5.preprocess(moc_headers: quazip_moc_headers)
+quazip_pp = qt.compile_moc(headers: quazip_moc_headers)
```

Comment:
I think switching from a non-deprecated method to a method new in 0.59 isn't really best for the WrapDB and we should offer broader support than that.